posthog/models/person.py

```diff
@@ -79,6 +79,22 @@ def merge_people(self, people_to_merge: List["Person"]):
     # Has an index on properties -> email, built concurrently
     # See migration 0121
 
+    @staticmethod
+    def get_distinct_ids_and_email_by_ids(person_ids, team_id):
```

Comment:
This seems to be the opposite of what it should be?

I.e., bulk querying uses the bulk API (which gains the advantage of querying in bulk), while the single query makes a call to the bulk end point with a list of one item.

https://docs.djangoproject.com/en/3.2/ref/models/querysets/#in-bulk - `in_bulk` achieves this, I think.

Reply:
This was a quick pass but I'll be refactoring with ejecting to SQL for best performance. Just trying to get it out the door quickly.